services-api/src/main/java/io/scalecube/services/methods/ServiceMethodInvoker.java

```diff
@@ -42,11 +51,13 @@ public ServiceMethodInvoker(Method method, Object service, MethodInfo methodInfo
    */
   public Mono<ServiceMessage> invokeOne(
       ServiceMessage message, BiFunction<ServiceMessage, Class<?>, ServiceMessage> dataDecoder) {
-    return Mono.defer(
+    Supplier<Mono<ServiceMessage>> invoke =
```

Comment:
is there more clean way to do it?

Reply:
This ugly `Supplier` is added only to avoid `checkstyle` alert regarding indentation.

Unfortunately, I didn't find a good solution for this part of the code to get rid of `Supplier` variable.

Reply:
I can move it to separate private method like:

```java
public Mono<ServiceMessage> invokeOne(
      ServiceMessage message, BiFunction<ServiceMessage, Class<?>, ServiceMessage> dataDecoder) {
    return Mono.defer(() -> invoke(message, dataDecoder))
        .onErrorResume(throwable -> Mono.just(errorMapper.toMessage(throwable)));
  }

  private Mono<ServiceMessage> invoke(
      ServiceMessage message, BiFunction<ServiceMessage, Class<?>, ServiceMessage> dataDecoder) {
    Object request = toRequest(message, dataDecoder);
    return Mono.from(invoke(request)).map(this::toResponse);
  }
```

Reply:
what is the checkstyle alert maybe we can remove it?

Reply:
> what is the checkstyle alert

Something like this:

```
'lambda arguments' has incorrect indentation level 16, expected level should be one of the following: 6, 8. (56:0) [IndentationCheck]
'block' child has incorrect indentation level 18, expected level should be one of the following: 8, 10, 12. (57:0) [IndentationCheck]
'block' child has incorrect indentation level 18, expected level should be one of the following: 8, 10, 12. (58:0) [IndentationCheck]
'block rcurly' has incorrect indentation level 16, expected level should be one of the following: 6, 8. (59:0) [IndentationCheck]
```